webstruct/html_tokenizer.py

```diff
@@ -202,7 +240,10 @@ def detokenize_single(self, html_tokens, tags):
         tree = html_tokens[0].root
 
         # find starts/ends of token groups
-        token_groups = self.sequence_encoder.group(zip(html_tokens, tags))
+        if self.bilou:
```

Comment:
With a proper implementation of BilouEncoder this condition won't be necessary, it'd be just `token_groups = self.sequence_encoder.group(zip(html_tokens, tags))`.